src/System.Net.HttpListener/tests/HttpListenerAuthenticationTests.cs

```diff
@@ -404,6 +424,24 @@ public void Realm_SetDisposed_ThrowsObjectDisposedException()
             return clientTask.Result;
         }
 
+        public async Task<HttpResponseMessage> AuthenticationFailureAsyncContext(HttpClient client, HttpStatusCode errorCode)
+        {
+            Task<HttpResponseMessage> clientTask = client.GetAsync(_factory.ListeningUrl);
+            Task<HttpListenerContext> serverTask = _listener.GetContextAsync();
+
+            // The client task should complete first - the server should send a 401 response.
```

Comment:
I think this is out of sync with the actual code that's checking for `errorCode`. I'll remove the comment: it's obvious that's what we expect from the assert below.